TROUBLESHOOTING.md

```diff
@@ -0,0 +1,43 @@
+# Troubleshooting
+
+This serves as a troubleshooting guide for any ts-jest related issues.
+
+## Running ts-jest on CI tools
+
+### PROBLEM
+
+Cannot find module "" from ""
+
+### SOLUTION
+
+- Check if `rootDir` is referenced correctly. If not add this on your existing jest configuration.
+
+```javascipt
+module.exports = {
+  ...
+   roots: ["<rootDir>"]
+}
+```
+
+- Check if module directories are included on your jest configuration. If not add this on your existing jest configuration.
+
+```javascipt
+module.exports = {
+  ...
+  moduleDirectories: ["node_modules","<module-directory>"],
+  modulePaths: ["<path-of-module>"],
+}
+```
+
+- Check if module name is properly mapped and can be referenced by jest. If not add this on your existing jest configuration.
+
+```javascipt
+module.exports = {
+  ...
+  moduleNameMapper: {
+    "^.*shared/(.*)": "<rootDir>/src/shared/$1",
```

Comment:
Can we use placeholder example here, something like `"<import-path>": "<rootDir>/<real-physical-path>"`

Reply:
Ok got it.